src/main/resources/static/css/cite.css

```diff
@@ -0,0 +1,17 @@
+body {
+    min-height: 75rem;
+    padding-top: 4.5rem;
+    padding-left: 10px;
+}
+label {
+    width: 180px;
+    clear: left;
+    text-align: left;
+}
+input, label {
+    float: left;
+}
+
+#item-status {
+    margin-bottom: 10px;
+}
```

Comment:
I discussed this in the previous Sprint, and it was suggested that a .css file was created to inject the styling to the HTML. This looked like it was on it's way to being completed but I went ahead and created it for you. This should resolve some technical debt for you going forward with new pages.

I also included a few styles to clean up the form, but it isn't necessary.